macros/EmbedTest262ReportResultsTable.ejs

```diff
@@ -0,0 +1,12 @@
+<%
+// Embeds a Test262.report results table
+//
+// Parameters:
+//  $0 - A "feature" name from test262.report
+//
+//  Example calls:
+// {{EmbedTest262ReportResultsTable("globalThis")}}
+// {{EmbedTest262ReportResultsTable("BigInt")}}
+//
+%>
+<iframe frameborder="0" scrolling="no" style="width: 100%; height: 300px;" src="https://test262.report/embed/features/<%-encodeURIComponent($0)%>?engines=chakra%2Cjavascriptcore%2Cspidermonkey%2Cv8&summary=true&include-browsers=true"></iframe>
```

Comment:
Currently on MDN, when bleaching the document content, we do not allow `scrolling` or `style` attributes on `iframe` elements (and I don't think we need the `frameborder` attribute?). When I tested this locally, I used:

```html
<iframe width="100%" height="300px" src="..."></iframe>
```

Also, MDN will strip-out any `iframe` sources that are not part of a whitelist (see https://github.com/mozilla/kuma/blob/8ac987263f711cb3662b01059cc8409f370c7775/kuma/settings/common.py#L1247), so I'll need to create a companion PR in Kuma to add `parse_iframe_url('https://test262.report/embed/features/')` to that whitelist in order for this to work.

Reply:
Thanks for the review @escattone. I'm a little confused by some of this feedback, as it appears to contradict existing macro conventions—maybe you can help me better understand :D 

> we do not allow `scrolling` or `style` attributes

- `style`: Is this restriction limited to `iframe`?
- `scrolling`: this appears in `EmbedNewRelicChart.ejs`—perhaps that predates the restriction? I can remove it

> `<iframe width="100%" height="300px" src="..."></iframe>`

Works for me, I will make this change. 

> Also, MDN will strip-out any iframe sources that are not part of a whitelist

Good to know—this hadn't been mentioned prior to now. I can make that PR if you'd like?

Thanks again!

Reply:
>     * `style`: Is this restriction limited to `iframe`?

Yes, that's correct.

>     * `scrolling`: this appears in `EmbedNewRelicChart.ejs`—perhaps that predates the restriction? I can remove it

Even though the `EmbedNewRelicChart.ejs` macro uses the `scrolling` attribute, it is removed by the bleaching process after the document is rendered, so there's nothing gained by using it in the macro.